pkg/get/get_test.go

```diff
@@ -6855,6 +6855,33 @@ func Test_Copacetic(t *testing.T) {
 
 }
 
+func Test_Crossplane(t *testing.T) {
+	tools := MakeTools()
+	name := "crossplane"
+
+	tool := getTool(name, tools)
+
+	const toolVersion = ""
+
+	test := test{
```

Comment:
Why wouldnt the test table cover all the available variations?
```
0              [darwin_amd64/](https://releases.crossplane.io/stable/v1.17.0/bin/darwin_amd64/)
0              [darwin_arm64/](https://releases.crossplane.io/stable/v1.17.0/bin/darwin_arm64/)
0              [linux_amd64/](https://releases.crossplane.io/stable/v1.17.0/bin/linux_amd64/)
0              [linux_arm/](https://releases.crossplane.io/stable/v1.17.0/bin/linux_arm/)
0              [linux_arm64/](https://releases.crossplane.io/stable/v1.17.0/bin/linux_arm64/)
0              [windows_amd64/](https://releases.crossplane.io/stable/v1.17.0/bin/windows_amd64/)
```

Reply:
done